apps/explorer/src/pages/address-result/AddressResult.tsx

```diff
@@ -13,15 +13,12 @@ import { OwnedCoins } from '~/components/OwnedCoins';
 import { OwnedObjects } from '~/components/OwnedObjects';
 import { PageHeader } from '~/ui/PageHeader';
 
-function AddressResultPageHeader({
-	address,
-	domainName,
-}: {
-	address: string;
-	domainName?: string;
-}) {
+function AddressResultPageHeader({ address, loading }: { address: string; loading?: boolean }) {
+	const { data: domainName, isFetching } = useResolveSuiNSName(address);
```

Comment:
using `isFetching` here instead of `isLoading` because we only run these queries on production, and `isLoading` is always `true` when the query is `disabled`

Ref: https://github.com/TanStack/query/issues/3584